pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -185,8 +185,20 @@ type StorageClusterStatus struct {
 
 	// ExternalSecretHash holds the checksum value of external secret data.
 	ExternalSecretHash string `json:"externalSecretHash,omitempty"`
+
+	// Images holds the image reconclie status for all images reconciled by the operator
+	Images map[string]ComponentImageStatusRef `json:"imageReconcileState,omitempty"`
```

Comment:
Something seems to be wrong with the CSV generation here, I think it doesn't like having a list of pointers. Unless there's any reason not to, can you update this to `Images []ComponentImageStatus`?

Reply:
fixed, not using maps anymore